src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -97,8 +97,8 @@ public override async Task<string> GetHelpTermAsync(Document document, TextSpan
             if (TryGetTextForSpecialCharacters(token, out var text) ||
                 TryGetTextForContextualKeyword(token, out text) ||
                 TryGetTextForCombinationKeyword(token, out text) ||
-                TryGetTextForKeyword(token, out text) ||
                 TryGetTextForPreProcessor(token, out text) ||
+                TryGetTextForKeyword(token, out text) ||
```

Comment:
have pp checks come first so we defer to the pp interepretation if inside a directive.